task.sh

```diff
@@ -0,0 +1,72 @@
+#   dir3
+#       dir4
+
+mkdir -p task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Флаг `-p` позволяет не указывать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```